types/underscore/index.d.ts

```diff
@@ -4881,6 +4924,18 @@ declare namespace _ {
          */
         compose(...functions: Function[]): Function;
 
+        /**
+         * Wrapped type `(...args: any[]) => any`.
+         * @see _.restArguments
+         */
+        restArguments(func:Function, startIndex:number): (...args: any[]) => any;
+
+        /**
+         * Wrapped type `Array<number | string>`.
+         * @see _.toPath
+         */
+        toPath(path: number | string | Array<number | string>): Array<number | string>;
```

Comment:
The first argument is implicit in the `Underscore` and `_Chain` interfaces.

第一个参数隐含在“Underscore”和“_Chain”接口中。

```suggestion
        /**
         * Wrapped type `(...args: any[]) => any`.
         * @see _.restArguments
         */
        restArguments(startIndex:number): (...args: any[]) => any;

        /**
         * Wrapped type `number | string | Array<number | string>`.
         * @see _.toPath
         */
        toPath(): Array<number | string>;
```